Assets/Scripts/Enviroment/EnvironmentSpawner.cs

```diff
@@ -14,7 +14,7 @@ public class EnvironmentSpawner : MonoBehaviour
     [SerializeField] float spawnAreaXMin;
     [SerializeField] float spawnAreaXMax;
     [SerializeField] float spawnAreaY;
-    public static float spawnCooldown;
+    [SerializeField] public static float spawnCooldown;
```

Comment:
```suggestion
    public static float spawnCooldown;
```
@pkuipers1 kan je alleen dit nog weg halen, is niet nodig.